org-theme/setup/long.setup

```diff
@@ -20,6 +20,6 @@
 # I recommend you to generate your own css/js files by forking, cloning and customizing the file
 # theme.json before building it.
 #
-#+HTML_HEAD: <link rel="stylesheet" type="text/css" href="/org-theme/dist/long/bundle.css"/>
-#+HTML_HEAD: <link rel="stylesheet" type="text/css" href="/css/syntax.css" />
-#+HTML_HEAD: <script src="/org-theme/dist/long/bundle.js" type="text/javascript" ></script>
+#+HTML_HEAD: <link rel="stylesheet" type="text/css" href="bundle.css"/>
```

Comment:
Why this path rename is necessary?

Reply:
We have to send a bundle (tar) to GitHub Pages.

We could either recreate this structure within the bundle or rename it.

It's easier to simply flatten all the static files and rename here, but I can change the approach.

Reply:
After you clarified, I think we can maintain this way. It's good enough.